sw/device/lib/dif/dif_kmac.h

```diff
@@ -172,6 +172,12 @@ typedef struct dif_kmac {
    */
   bool squeezing;
 
+  /**
+   * Whether the output length (d) in bits should be right encoded and appended
+   * to the end of the message.
```

Comment:
nit: Can you add the modes where this is required?

Reply:
Done (just KMAC).